spec/contracts.dd

```diff
@@ -75,6 +75,13 @@ do
     ...code...
 }
 ------
+
+    $(P The actual function body starts with $(D do).
+    In the past, $(D body) was used, and could still be encountered in old code bases.
+    In the long term, $(D body) may be deprecated, but for now it's still allowed
```

Comment:
Remove 'still'

Reply:
Why don't you state that it will be deprecated. After all, the regarding DIP has been approved.

Reply:
Yeah, but the problem is not the compiler change, the problem is the documentation that has been brutally changed without a changelog entry, that's the point of the bugzilla report. Even in phobos you still use `body`.